formicaio/docker-compose.yml

```
@@ -0,0 +1,29 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: formicaio_formicaio_proxy_1
```

After adapting the name of the service, change the name to the following.
```suggestion
      APP_HOST: formicaio_proxy_1
```